File: angular/src/app/main/supplyChain/sales/shared/dtos/oeRoutes-dto.ts
export class oeroutesdto {
    id:number;
    routID:number;
    routDesc:string;
    active: boolean;
    createdBy: string;
    createDate: string | Date;
    audtUser: string;
    audtDate: string | Date;
}
export class GetoeroutesDtoOutput {
    oeRoutesDto: oeroutesdto
}

export class PagedResultDtocader {
    totalCount: number;
    items: GetoeroutesDtoOutput[]
}

File: aspnet-core/src/ERP.Application.Shared/SupplyChain/Sales/OERoutes/Dtos/OERoutesDto.cs
using System;
using Abp.Application.Services.Dto;

namespace ERP.SupplyChain.Sales.OERoutes.Dtos
{
    public class OERoutesDto : EntityDto
    {
        public int RoutID { get; set; }

        public string RoutDesc { get; set; }

        public bool Active { get; set; }

        public string CreatedBy { get; set; }

        public DateTime? CreateDate { get; set; }

        public string AudtUser { get; set; }

        public DateTime? AudtDate { get; set; }

    }
}